pkg/kubelet/config/config.go

```diff
@@ -265,6 +272,7 @@ func (s *podStorage) merge(source string, change interface{}) (adds, updates, de
 				ref.Annotations = make(map[string]string)
 			}
 			ref.Annotations[kubelet.ConfigSourceAnnotationKey] = source
+			recordFirstSeenTime(ref)
```

Comment:
Yes, if we go with that, I can set the StartTime here, and probably get rid of the annotation completely. I *think* that should work. I went for the annotation only because I didn't want to add any additional field to the pod.